File: C_Programming/Competitive_Programming/Assignment3/program2.c
// Write a program which accept one number from user and print even factors of that number

//Input : 24
//Output : 1 2 4 6 8 12
#include<stdio.h>

void DisplayFactor(int iNo) 
{
    int iCnt = 0;

    if(iNo <= 0)
    {
        iNo = -iNo;
    }

    for(iCnt = 1; iCnt < iNo; iCnt++)
        {
            if((iNo % iCnt) == 0)
            {
                printf("%d\t",iCnt);
            }
        }

}

int main()
{
    int iValue = 0;

    printf("Enter number : ");
    scanf("%d",&iValue);

    DisplayFactor(iValue);

    return 0;
}

File: Java_Programming/Competitive_Programming/Assignment18/program2.java
// Write a program to print all even numbers up to N

class logic 
{
    void printEvenNumber(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if((iCnt % 2) == 0)
            {
                System.out.print(iCnt+"\t");
            }
        }

    }
}

class program2
{
    public static void main(String A[] ) 
    {
        logic obj = new logic();
        obj.printEvenNumber(20);
    }
}
